src/yz_solr.erl

```diff
@@ -279,8 +279,11 @@ prepare_json(Docs) ->
 -spec partition_list(index_name()) -> {ok, Resp::binary()} | {error, term()}.
 partition_list(Core) ->
     Params = [{q, "*:*"},
+              {rows, 0},
+              {omitHeader, <<"true">>},
               {facet, "on"},
               {"facet.mincount", "1"},
+              {"facet.method", <<"enum">>},
```

Comment:
I shouldn't matter. The `omitHeader` line was actually taken from https://github.com/basho/yokozuna/blob/95a03badd73d4b262ca68571b222279ade1d1c77/src/yokozuna.erl#L143 (to keep some consistency in the code base). I didn't change the other occurences of plain strings to avoid too much code churn. Yet another variant exists at https://github.com/basho/yokozuna/blob/95a03badd73d4b262ca68571b222279ade1d1c77/src/yz_pb_search.erl#L154, where atoms are used.

I propose to follow this up with a separate patch that introduces consistent usage of one of these forms.